src/zarr/api/synchronous.py

```diff
@@ -895,6 +902,212 @@ def create_array(
                 storage_options=storage_options,
                 overwrite=overwrite,
                 config=config,
+                write_data=write_data,
+            )
+        )
+    )
+
+
+def from_array(
+    data: Array | npt.ArrayLike,
+    store: str | StoreLike,
+    write_data: bool = True,
+    *,
+    name: str | None = None,
+    chunks: Literal["auto", "keep"] | ChunkCoords = "keep",
+    shards: ShardsLike | None = None,
+    filters: FiltersLike | Literal["keep"] = "keep",
+    compressors: CompressorsLike | Literal["keep"] = "keep",
+    serializer: SerializerLike | Literal["keep"] = "keep",
+    fill_value: Any | None = None,
+    order: MemoryOrder | None = None,
+    zarr_format: ZarrFormat | None = None,
+    attributes: dict[str, JSON] | None = None,
+    chunk_key_encoding: ChunkKeyEncoding | ChunkKeyEncodingLike | None = None,
```

Comment:
```suggestion
    chunk_key_encoding: ChunkKeyEncodingLike | None = None,
```
`ChunkKeyEncodingLike` includes `ChunkKeyEncoding`